modules/matchPattern.js

```diff
@@ -1,21 +1,30 @@
 import pathToRegexp from 'path-to-regexp'
 
 const cache = {}
+const pathToRegexpOptions = {strict: true}
```

Comment:
Will do. Pulling the constant out was a nano-optimization. Sorry about not checking the coding style.